ecr_api.py

```diff
@@ -440,7 +723,8 @@ def handle_invalid_usage(error):
 app.add_url_rule('/healthy', view_func=Healthy.as_view('healthy'))
 app.add_url_rule('/apps', view_func=AppList.as_view('appsListAPI'))
 app.add_url_rule('/apps/<string:app_id>', view_func=Apps.as_view('appsAPI'))
-
+#app.add_url_rule('/permissions/<string:app_id>', view_func=Permissions.as_view('permissionsAPI'))
+app.add_url_rule('/apps/<string:app_id>/permissions', view_func=Permissions.as_view('permissionsAPI'))
 
 if __name__ == '__main__':
     app.run(debug=True, host='0.0.0.0')
```

Comment:
You can add a #pragma here as you are never going to test this line

`app.run(debug=True, host='0.0.0.0') # pragma: no cover`